src/data/User.js

```diff
@@ -816,39 +828,18 @@ const Users = {
       portfolio: "",
     },
   },
-  "leander006": {
+    "leander006": {
     name: "Leander D'silva",
     branch: "IT",
-    description: "Exploring blockchain and expanding my knowledge by learning innovative concepts",
+    description:
+      "Exploring blockchain and expanding my knowledge by learning innovative concepts",
     links: {
       github: "https://github.com/leander006",
       instagram: "https://www.instagram.com/leander_dsilva06",
       linkedin: "https://www.linkedin.com/in/leander06",
       portfolio: "https://leanderdsilva.netlify.app",
     },
   },
-  "pratikwadke02": {
```

Comment:
@0xajinkya it seems like you removed some names from the bottom :( ... Also could you append your name at the last instead of appending it at very beggining